Assets/GothicVR/Scripts/Caches/AssetCache.cs

```diff
@@ -62,6 +63,37 @@ public Texture2D TryGetTexture(string key)
             return texture;
         }
 
+        public async Task<Texture2D> TryGetTextureAsync(string key)
```

Comment:
Uh really? Why do we need to start with async stuff? If needed, we should rethink where to put the async calls.